R/community.R

```diff
@@ -696,15 +696,15 @@ as.hclust.communities <- function(x, hang = -1, use.modularity = FALSE,
 #' @rdname communities
 #' @family community
 #' @export
-as_phylo <- function(x, ...) {
-  UseMethod("as_phylo")
+as.phylo <- function(x, ...) {
```

Comment:
There is already `ape::as.phylo()`, do we really need to do anything here?

Reply:
Should this just be deleted? As noted in the other comment I'm now confused.

Reply:
Yes, I guess we don't need it here.